pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "poem-plugins"
-version = "0.12.12"
+version = "0.14.0.dev1"
```

Comment:
It seems out that the long version can be written not through `+` but through a dot, or even from the config value

Reply:
```Build metadata MUST be ignored when determining version precedence. Thus two versions that differ only in the build metadata, have the same precedence.```

Patch is always increase, seems ok?